### ISSUE TITLE
Click event not being propagated inside a section using scrollOverflow:true

### ISSUE THREAD

**Issue by sjaakbanaan** [alvarotrigo/fullPage.js#2047]
Hi,

I was very pleased to see that slimScroll got replaced by iScroll, but I discovered a bug in combination with my custom js code that was working in previous versions of fullPage:

I created a codepen with the issue:
http://codepen.io/sjaakbanaan/pen/GqgZjK

**The bug:**
Whenever de content in the div that get's shown on click is bigger then the current section, i can't close it anymore and can't open another one.

**How to recreate:**
- use a desktop with a screen > 1680 x 1080 (the preview of the pen needs to be at least 500 px high);
- scroll to the second section and click on the first 'some question' link. No issues whatsoever because the shown content still fits in the section without creating a scrollbar;
- now refresh and open the second 'some question' link. Now a scrollbar gets added and breaks the show/hide functionality completely.

**Comment by sjaakbanaan** [comment]
Found a fix! Not really a bug though, more a weird default config setting. I  had to add:

`scrollOverflowOptions: {
			click: false
}`

even though 'false' is the default setting for 'click' in iScroll.

**Comment by alvarotrigo** [comment]
Related with this https://github.com/alvarotrigo/fullPage.js/issues/2035

**Comment by alvarotrigo** [comment]
Fixed in fullpage.js 2.8.2

**Comment by shanriyan** [comment]
Hi,
I am very happy to use fullpage.js. 
I know there is the $.fn.fullpage.silentMoveTo('anchor-link'); function and I am trying to use it in this way but it doesn't work:
  onLeave: function(index, nextIndex, direction,anchorLink ){
                var isDown=direction=="down"?true:false;
                 //Section 1
                if(index == 1 && isDown){
                    $.fn.fullpage.silentMoveTo("Section10");
});
Error displayed
![image](https://cloud.githubusercontent.com/assets/29113630/26665462/6b6a339c-46b7-11e7-9ef8-f669335009d0.png)

**Comment by nnmrts** [comment]
Since the bug with `scrollOverflowOptions: { click: false }` was an iScroll issue you fixed in your customized `0.0.1` version, could you please include this updated scrolloverflow.js file in the bower release? Spent 19 hours on figuring out an issue with not being able click things in my webapp, including checking this issue on github and then checking the version of fullpagejs in my bower_components. There it says 2.9.4. Well no, it isn't 2.9.4. It is 2.9.4.fuckitijustpretendtobethatversionbutimactually2.8.1becausetheretheiscrollissueisntfixed

Sorry. Really, I'm sorry, but as we are both developers, I hope you understand my frustration and fix this sometime.

Anyway, thanks for your work. I love this "plugin", which in fact isn't just a plugin, it's a whole library of awesome things.

💓

PS: Somehow your tag `2.9.4` isn't equal with the master branch. At least not for the scrolloverflow.js file. However, the current release is 2.9.4, so it should be. My assumption is that bower checks your library by the tags and then gets the files from there.

You probably just have to do this:
```
git checkout master
git merge 2.9.4
```
or

```
git checkout master
git merge 2.8.1
git merge 2.8.2
git merge ...
```
and so on all the way up.

**Comment by alvarotrigo** [comment]
@nnmrts wow, didn't realise!! Thanks for reporting it! I'll create a new issue for it!
Please check this topic for updates https://github.com/alvarotrigo/fullPage.js/issues/2868
